pyanaconda/startup_utils.py

```diff
@@ -71,8 +92,365 @@ def get_anaconda_version_string():
         # Ignore pylint not finding the version module, since thanks to automake
         # there's a good chance that version.py is not in the same directory as
         # the rest of pyanaconda.
-        from pyanaconda import version # pylint: disable=no-name-in-module
+        from pyanaconda import version  # pylint: disable=no-name-in-module
```

Comment:
Just a question, is this intentional? I looks to me like typo more than correction.